src/avram/query_builder.cr

```diff
@@ -254,6 +254,12 @@ class Avram::QueryBuilder
       .map(&.split('.').last)
   end
 
+  # TODO: rename/refactor
+  def select_direct(selection : Array(ColumnName))
```

Comment:
Could use some help with naming/using existing methods to do this

The reason select(Array(ColumnName)) [right below] doesn't work is because it re-prefixes the @table and our groups from previous method have the table already included in the selection, as well as COUNT(*) needing no table prefix.

Reply:
I think `select_direct` sounds fine. `unscoped_select` could be another one... Chances are this will be more of an escape hatch than a direct API usage.

Reply:
`select_direct` is opaque to me. This returns a list of columns, can it just be `selected_columns`?

Reply:
Maybe we need to see why using the normal `select` method prepends the table twice.... I think that was the reason for this second method anyway, right?

`query.select(query.groups)` was causing something like `GROUP BY table.table.column` or something 🤔 so we had to come up with a different method that didn't pre-scope the columns. I think this method is more of a hack, and really shouldn't be exposed externally. I just forget why it was needed.

Reply:
https://github.com/luckyframework/avram/pull/778/files#diff-df659060f00cb38deae2b7b48c5172fa24e3fd5b7f3652d35f96f992b35b0116R263
select itself prepends the table name

Reply:
One option might be to have a `prepend : Boolean = true` default arg on the select function itself